spec/services/publishing_api_payload_spec.rb

```diff
@@ -49,6 +49,13 @@
       expect(payload).not_to include("first_published_at")
     end
 
+    it "specifies organistions when the edition is access limited" do
+      edition = build(:edition, :access_limited)
+      allow(edition).to receive(:access_limit_organisation_ids) { %w[org-id] }
+      payload = PublishingApiPayload.new(edition).payload
+      expect(payload["access_limited"]["organisations"]).to eq %w[org-id]
+    end
+
```

Comment:
How do we represent access_limited in the payload when there is no access_limits?

Reply:
Yup. I've added one now. The lack of coverage for `auth_bypass_ids` anywhere distracted me; the new test covers this and implicitly covers the org behaviour without access limits.